Examples/Modules/qed/schwinger/analysis.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2020 Luca Fedeli, Neil Zaim
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+import os
+import glob
+import yt
+import numpy as np
+
+# define some parameters
+
+c = 299792458.
+m_e = 9.10938356e-31
+e = 1.6021766208e-19
+hbar = 1.054571800e-34
```

Comment:
The problem is that the second test is extremely sensitive and fails if I use scipy constants (which are slightly different from the ones in picsar).

This might not be ideal, especially if we someday change the constants in picsar (don't know why we would do that) or if we want to make Schwinger work in single precision (which may result in the second test failing). If you think it's better I can reduce the sensitivity of the second test by reducing the number of pairs created per cell.

Reply:
Ah! The discrepancy is due to the fact that constants in PICSAR come from CODATA 2014, while scipy (and WarpX) use newer data (CODATA 2018). 
I think that we could leave the hard-coded constants, but we should also consider to update the values of the fundamental constants in PICSAR!